src/System.Data.SqlClient/tests/ManualTests/SQL/UdtTest/UdtTest.cs

```diff
@@ -40,6 +40,28 @@ public static void GetSchemaTableTest()
         }
 
         [CheckConnStrSetupFact]
+        public static void GetValueTest()
+        {
+            using (SqlConnection conn = new SqlConnection(DataTestUtility.TcpConnStr))
```

Comment:
`using` statement does not need block? I don't see opening and ending brackets.

Reply:
You can chain them together like this to avoid using lots of curly braces.